lib/hammer_cli/defaults.rb

```diff
@@ -100,9 +100,13 @@ def switch_to_name(opt)
   end
 
   def self.defaults
-    @defaults ||= Defaults.new(HammerCLI::Settings.settings[:defaults])
-
+    return @defaults if @defaults
+    settings = HammerCLI::Settings
+    default_values = settings.get(:use_defaults) ? settings.get(:defaults) : {}
+    @defaults = Defaults.new(default_values)
```

Comment:
Maybe it's just me, but this seems difficult to read.
How about:
```ruby
@defaults ||= Defaults.new(default_values_from_settings)

private
def default_values_from_settings
  HammerCLI::Settings.get(:use_defaults) ? HammerCLI::Settings.get(:defaults) : {}
end
```

Reply:
@tstrachota it's a way better! You were awarded by the `rubocop of the day` trophy. :robot: